src/services/rabbitmq/queues/AdminQueue.ts

```diff
@@ -35,16 +33,16 @@ export class AdminQueue extends RabbitMQQueue implements PS2AlertsQueueInterface
             setup: async (channel: ConfirmChannel) => {
                 await Promise.all([
                     channel.checkExchange(config.rabbitmq.exchange),
-                    channel.assertQueue(this.thisQueueName, queueOptions),
+                    channel.assertQueue(this.queueName, queueOptions),
```

Comment:
Oh, no, I saw what you did there and it was because of the fact that `queueName` was private. I have changed it to protected in the parent so it is accessible by children.

Reply:
I tried to use protected but I couldn't because the abstract demands it's set, and we want to override it in the child class, I forget the exact logical reasoning why I couldn't just have the queue name in the child hence this weirdness

Reply:
I see. What I can say for sure is that this works. The reason is that during compilation it didn't throw an error and as it runs as JS `queueName` will be available whether it is public, protected, or private anyway as that that is only part of TS.